EditPortfolio.js

```diff
@@ -7,8 +7,23 @@ const inputCompanyName = document.getElementById("companyName");
 const inputStartDate = document.getElementById("startDate");
 const inputEndDate = document.getElementById("endDate");
 const inputDescription = document.getElementById("description");
+const apiUrl = "http://localhost:5501/api/portfolio/experiences";
```

Comment:
Please create an environment file (.env) and put base url there. Use Base URL from .env file all over the project

Reply:
put this URL "http://localhost:5501/api" to .env file

Reply:
and Please don't resolve any conversation ... I will do this after checking